_data/navigation.yaml

```diff
@@ -165,8 +165,8 @@ docs_nav:
         url: /pages/configs/deploy_json
       - title: experiments.json
         url: /pages/configs/experiments_json
-      - title: pnpmfile.js
-        url: /pages/configs/pnpmfile_js
+      - title: .pnpmfile.cjs
+        url: /pages/configs/pnpmfile_cjs
```

Comment:
@kebapBoy thanks for making this PR!  We overlooked your PR because the website has been moved to a new monorepo.

I've reapplied your fix in this PR in the new repo: https://github.com/microsoft/rushstack-websites/pull/82